src/frontends/paddle/tests/test_models/gen_scripts/generate_embedding.py

```diff
@@ -98,73 +104,85 @@ def embedding(name : str, ids, vocab_size, embedding_dim, padding_idx=None, spar
-    embedding("embedding_sparse", ids, vocab_size, embedding_dim, sparse=True, vocab_embeddings=table, compare=False)    
+    embedding("embedding_sparse", ids, vocab_size, embedding_dim,
+              sparse=True, vocab_embeddings=table, compare=False)
 
     # # compare fail
     ids = np.random.randint(0, vocab_size, 4).astype("int32")
-    embedding("embedding_none_weight", ids, vocab_size, embedding_dim, compare=False)
+    embedding("embedding_none_weight", ids,
+              vocab_size, embedding_dim, compare=False)
 
     #
     ids = np.random.randint(0, vocab_size, 4).astype("int32")
     ids = np.squeeze(ids)
     padding_idx = np.random.choice(ids, 1)[0]
     # print('padding_idx {}, ids {}'.format(padding_idx, ids))
-    outputs = embedding("embedding_paddings", ids, vocab_size, embedding_dim, padding_idx=int(padding_idx), vocab_embeddings=table, compare=False)
+    outputs = embedding("embedding_paddings", ids, vocab_size, embedding_dim, padding_idx=int(
+        padding_idx), vocab_embeddings=table, compare=False)
     # print('outputs {}'.format(outputs))
 
     # corner case
     ids = np.random.randint(0, vocab_size, 4).astype("int32")
-    pick = np.random.choice(4, 1)[0] # pick randomly to be max vacab_size -1
-    ids[pick] = vocab_size-1
+    pick = np.random.choice(4, 1)[0]  # pick randomly to be max vacab_size -1
+    ids[pick] = vocab_size - 1
     padding_idx = -1
     # print('padding_idx {}, ids {}'.format(padding_idx, ids))
-    outputs = embedding("embedding_paddings_neg1", ids, vocab_size, embedding_dim, padding_idx=int(padding_idx), vocab_embeddings=table, compare=False)
-    # print('outputs {}'.format(outputs))    
+    outputs = embedding("embedding_paddings_neg1", ids, vocab_size, embedding_dim,
+                        padding_idx=int(padding_idx), vocab_embeddings=table, compare=False)
+    # print('outputs {}'.format(outputs))
 
     #
-    ids = np.random.randint(low=0, high=vocab_size, size=(2, 4, 5)).astype("int32")
-    embedding("embedding_tensorIds", ids, vocab_size, embedding_dim, vocab_embeddings=table, compare=False)
-    
+    ids = np.random.randint(low=0, high=vocab_size,
+                            size=(2, 4, 5)).astype("int32")
+    embedding("embedding_tensorIds", ids, vocab_size,
+              embedding_dim, vocab_embeddings=table, compare=False)
+
     #
-    ids = np.random.randint(low=0, high=vocab_size, size=(2, 4, 5)).astype("int32")
+    ids = np.random.randint(low=0, high=vocab_size,
+                            size=(2, 4, 5)).astype("int32")
     flatten_idx = ids.flatten()
     padding_idx = np.random.choice(flatten_idx, 1)[0]
     # print('padding_idx {}'.format(padding_idx))
-    outputs = embedding("embedding_tensorIds_paddings", ids, vocab_size, embedding_dim, padding_idx=cpt.long_type(padding_idx), vocab_embeddings=table, compare=False)
-    # print('outputs {}'.format(outputs))
-   
+
+    if paddle.__version__ >= '2.0.0':
```

Comment:
This script can't run. Have the error below.
`Traceback (most recent call last):
  File "/home/iot/tmp/openvino/src/frontends/paddle/tests/test_models/gen_scripts/generate_embedding.py", line 182, in <module>
    if paddle.__version__ >= '2.0.0':
NameError: name 'paddle' is not defined`